katz/src/test/kotlin/laws/FunctorLaws.kt

```diff
@@ -0,0 +1,32 @@
+package katz
+
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+
+object FunctorLaws {
+
+    inline fun <reified F> laws(AP: Applicative<F> = applicative<F>()): List<Law> =
```

Comment:
We can't create instances in the generator unless we create a specific typeclass for `pure` or `apply` since that is defined in Applicative and that is what we need to lift values into `F[_]`